cli/cmd/inject.go

```diff
@@ -243,7 +243,7 @@ func injectPodTemplateSpec(t *v1.PodTemplateSpec) enhancedPodTemplateSpec {
 			},
 		},
 		Env: []v1.EnvVar{
-			v1.EnvVar{Name: "CONDUIT_PROXY_LOG", Value: "trace,h2=debug,mio=info,tokio_core=info"},
+			v1.EnvVar{Name: "CONDUIT_PROXY_LOG", Value: "warn,conduit_proxy=info"},
```

Comment:
Down the road we could make this configurable via a flag to the inject subcommand, but definitely doesn't have to happen as part of this branch.